packages/components/docs/sass.md

```diff
@@ -12430,19 +12500,57 @@ File uploader styles
   }
 
   .#{$prefix}--file__state-container .#{$prefix}--file-complete {
-    fill: $support-02;
+    fill: $interactive-04;
     cursor: pointer;
 
     &:focus {
       @include focus-outline('border');
     }
+
+    // for checkmark contrast
+    [data-icon-path='inner-path'] {
+      opacity: 1;
+      fill: $ibm-color__white-0;
```

Comment:
Just wanted to double-check if we want to use a theme token here.

Reply:
> Dark themes, success icon's inside checkmark should be a white fill. Also success icon's blue color should be coming from token interactive-04, which would have Blue50 for dark themes.

this change was based on this review point. I don't think we have a token for this so I am hard coding white the value cc @shixiedesign